internal/quaiapi/api.go

```diff
@@ -989,8 +987,7 @@ type RPCTransaction struct {
 	BlockHash         *common.Hash             `json:"blockHash"`
 	BlockNumber       *hexutil.Big             `json:"blockNumber"`
 	From              *common.MixedcaseAddress `json:"from,omitempty"`
-	Gas               hexutil.Uint64           `json:"gas"`
-	MinerTip          *hexutil.Big             `json:"minerTip,omitempty"`
+	Gas               hexutil.Uint64           `json:"gas,omitempty"`
```

Comment:
Omitting this causes RPC response issues in my experience, would rather it just be zero than not there at all

Reply:
mistake, changed